qa/Gemfile

```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+gem "runner-tool", :git => "https://github.com/purbon/runner-tool.git"
+gem "rspec", "~> 3.1.0"
+gem "rake"
+gem "pry", :group => :test
+gem "stud"
+gem "logstash-devutils"
```

Comment:
Do we need to be more version specific here?  We're pretty particular about it in other Gemfiles, so I am just asking.